server/metadata_context.go

```diff
@@ -56,6 +57,11 @@ func encodeExtAuthzMetadata(api string, ac *auth.Context, authorized bool) *stru
 		headerOrganization:   stringValueFrom(ac.Organization()),
 		headerScope:          stringValueFrom(strings.Join(ac.Scopes, " ")),
 	}
+
+	if ac.CustomAttributes != "" {
```

Comment:
Just one more quick thing: Could you please add a test to verify that this header isn't added when there is no value?